src/helper/chains.ts

```diff
@@ -0,0 +1,42 @@
+import { ChainDetails } from "../interface/interface.js";
+
+// List of supported chains in zkwasm service
+// TODO: Might need to add these server-side and directly send to client, or clients can implement their own chain details
+// as long as they are compatible with the zkwasm service.
+// Currently using ChainInfo type but no public RPC url is provided there.
+// Would most of the time be used to add a chain to the wallet.
+export const GoerliChainInfo: ChainDetails = {
```

Comment:
We definitely should not do hard code like this in service helper.
By review codes, I think you are just trying to resolve the issue that if network is not setting up in metamask, we need info to add the network?

If I am correct, we should use the solution which we used before for that zkcross project:

When the main page loading, it will check whether the browser has metamask and whether it has the network settings for the network id's which match our B/E server, if not (means no such network set up), it will add the public network settings based on the config in B/E server wallet-secret file.

It can be seperate item if it is complex.
In our playground code, yes, we can move switchnet to service helper browser provider, but it should not rely on metatask. So if it failed, just show something like "failed, please manually add this XXX network to metamask"

Please note, the server B/E server_config file may change and the network settings may be different, so we should not rely on any codes in service helper. 

Everything should only rely on the playground (caller app's config), lib should not such things. Thus in this lib, if network is not setting up in metamask, we should return error sometting like "Switch network to XXX failed, please add it to metamask". The caller app should be responsible to ensure the network is added to metamask, or just show error.